library/core/registry/src/main/java/org/quiltmc/qsl/registry/impl/RegistryConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.registry.impl;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.quiltmc.config.api.WrappedConfig;
+import org.quiltmc.config.api.annotations.Comment;
+import org.quiltmc.loader.api.config.QuiltConfig;
+
+@ApiStatus.Internal
+public class RegistryConfig extends WrappedConfig {
+	public static final RegistryConfig INSTANCE = QuiltConfig.create("quilt", "registry", RegistryConfig.class);
```

Comment:
Let's define a standard here already: `qsl` as the containing folder, module name for the relevant config;
This is what Key Binds API was using, and it would be better in the long-run
```suggestion
	public static final RegistryConfig INSTANCE = QuiltConfig.create("qsl", "registry_sync", RegistryConfig.class);
```

Reply:
I would keep the containing folder as quilt.

Reply:
Once again, `quilt` is Loader (or perhaps even `quilt_loader`), `qsl` is QSL; We shouldn't repeat the whole "Use `fabric` as Fabric API's mod ID" mistake

Reply:
This is configuration though, very different from a mod id.

My issue with this is if someone who doesn't know what QSL is (which could happen once there's dependency downloading yada yada) it would confuse people.

Reply:
It's the same fundamental issue: it tries to simplify too much, and in the result, everyone loses;
There was the whole "We'll call QFAPI "QSL" in order to confuse less people", but in the end, everyone was pretty confused when "QSL" turned out to be QFAPI bundling QSL; We shouldn't lie to the user here, let them know about `qsl`

Reply:
Moved it under `quilt/qsl/registry.toml`. Used `registry` just in case of config being extended in future for non-regsync related options.